BookingsAPI/Bookings.API/Controllers/HearingsController.cs

```diff
@@ -383,6 +382,20 @@ public async Task<ActionResult<BookingsResponse>> GetHearingsByTypes([FromQuery(
             return Ok(response);
         }
 
+        /// <summary>
+        /// Anonymises the Hearings, Case, Person and Participant data.
+        /// </summary>
+        /// <returns></returns>
+        [HttpPatch("anonymisehearings")]
+        [SwaggerOperation(OperationId = "AnonymiseHearings")]
+        [ProducesResponseType((int)HttpStatusCode.NoContent)]
+        public async Task<IActionResult> AnonymiseHearingsAsync()
+        {
+            var anonymiseConferenceCommand = new AnonymiseHearingsCommand();
```

Comment:
Should rename this variable to hearingscommad not conference Lionel. Thanks

Reply:
renamed variable